.vscode/settings.json

```diff
@@ -0,0 +1,26 @@
+{
+  // hide cache files
+  "files.exclude": {
+    "**/__pycache__": true,
+    "**/.mypy_cache": true,
+    "**/.pytest_cache": true,
+  },
+  // brunette & black
+  "editor.formatOnSave": true,
+  "python.formatting.provider": "black",
+  "python.formatting.blackPath": "brunette",
+  // isort
```

Comment:
just referring to the doc here: https://github.com/SatelCreative/skr/blob/main/python/python_vscode.md
maybe adding following makes it more specifically use `isort`?
```json
    "python.sortImports.args": [
        "-sp .isort.cfg"
    ],
```

Reply:
We actually don't have an isort.cfg. If seems to be detecting the config for me - maybe on windows it isn't working?

Reply:
Right, we don't have an `isort.cfg` file. 😅  I use WSL for `spylib`, it automatically sorts the import for me on save. 
I guess for the future, whether to standardize the isort settings as well. e.g. used in the [satel-webapp-boilerplate](https://github.com/SatelCreative/satel-webapp-boilerplate/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/server/.isort.cfg) [multi_line_output = 3](https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html#3-vertical-hanging-indent)